src/json/federate-role_info.json

```diff
@@ -129,5 +129,26 @@
          "Resource": "*"
         }
         ]
-    }
+    },
+	"crawsPolicies": [{
```

Comment:
We may want to include a customer managed inline policy document here instead of using AWS managed policies.